solidity/random-beacon/contracts/libraries/Relay.sol

```diff
@@ -131,15 +131,10 @@ library Relay {
         Data storage self,
         bytes calldata entry,
         bytes storage groupPubKey
-    ) internal returns (uint256 slashingAmount) {
+    ) internal returns (uint256) {
```

Comment:
Could we return a `uint96` value? This is expected by [`TokenStaking.seize`](https://github.com/threshold-network/solidity-contracts/blob/702aa5f12242e885e7b0c2b3228bb788439d30a8/contracts/staking/TokenStaking.sol#L1125) and [`TokenStaking.slash`](https://github.com/threshold-network/solidity-contracts/blob/702aa5f12242e885e7b0c2b3228bb788439d30a8/contracts/staking/TokenStaking.sol#L1110) functions.

Reply:
yea, we can do here. It will add some gas, but it needs to be done somewhere anyway [221a3ce](https://github.com/keep-network/keep-core/pull/2840/commits/221a3ceac74de426ff4c22e55b1c08e007da3405)